source/_integrations/modbus.markdown

```diff
@@ -198,6 +198,19 @@ Description:
 | value     | (write_register) A single value or an array of 16-bit values. Single value will call modbus function code 0x06. Array will call modbus function code 0x10. Values might need reverse ordering. E.g., to set 0x0004 you might need to set `[4,0]`, this depend on the byte order of your CPU |
 | state     | (write_coil) A single boolean or an array of booleans. Single boolean will call modbus function code 0x05. Array will call modbus function code 0x0F |
 
+### Example: writing a float32 type register
+
+To write a float32 datatype register use network format like `10.0` == `0x41200000` (network order float hexadecimal). Call the modbus.write_register:
```

Comment:
```suggestion
To write a float32 datatype register use network format like `10.0` == `0x41200000` (network order float hexadecimal). Call the `modbus.write_register` service:
```

Reply:
I would prefer:
```suggestion
To write a float32 datatype register use network format like `10.0` == `0x41200000` (network order float hexadecimal). Do:
```
Because the content says directly it is a service call.